testsuite/oidc/rhsso/__init__.py

```diff
@@ -63,7 +63,8 @@ def commit(self):
             publicClient=False,
             protocol="openid-connect",
             standardFlowEnabled=False)
-        self.user = self.realm.create_user(self.test_username, self.test_password)
+        user_id = self.realm.create_user(self.test_username, self.test_password)
```

Comment:
what if `create_user` method would return already initialized `User` class directly instead of his id?